pepper-apis/config/DataDonationPlatform.yaml.ctmpl

```diff
@@ -0,0 +1,25 @@
+{{with $environment := env "ENVIRONMENT"}}
+service: pepper-backend
+runtime: java11
+instance_class: B4
+{{if eq $environment "prod"}}
+basic_scaling:
+  max_instances: 6
+  idle_timeout: 45m
+{{else}}
+basic_scaling:
+  max_instances: 4
+  idle_timeout: 20m
+{{end}}
+
```

Comment:
More conservative and generous settings for prod than in other environments.
Stretched out the idle_timeout (we were using 10m before) as even during a workday it appear/disappear multiple times a day with just 10min.